### DISCUSSION TITLE
1.4.2

### DISCUSSION THREAD

**Discussion by louislam** [louislam/dockge#393] [Announcements]

**Comment by aenaveen** [comment]
Will there be an update for dockge? It has been a year now? Will there be any development on this project?

**Reply by basnijholt** [reply]
Wondering the same. I love this project though!

**Comment by StefanFlaschko** [comment]
@louislam Any Updates planned for dockge?  Would be highly appreciated.

**Comment by astappiev** [comment]
Maybe at least a minor version can be released with a few important patches, like pasting into the terminal?
